### DISCUSSION TITLE
Show and tell: my metrics dashboard

### DISCUSSION THREAD

**Discussion by qu8n** [tremorlabs/tremor-npm#319] [Show and tell]
Absolutely loving Tremor so far. Your project simplified the process of building a dashboard for me, and I appreciate how well-written and intuitive the documentations were written.

Wanted to share this dashboard that I've been working on: https://www.focusbeacon.com/dashboard/demo

**Comment by severinlandolt** [comment]
Very nice analytics, congratulations!👏🙌

**Comment by teamsaasbox** [comment]
Hi @qu8n, looks good. Did you build the navbar from scratch or is it taken from another project like tailwindui? Is your github project open source? https://github.com/qu8n/focusbeacon/blob/main/src/components/NavBar.js

I need a react navbar component with user profile dropdown to demonstrate our tool, but it has to be open source so we can distribute on github.
